lib/config.js

```diff
@@ -42,8 +43,15 @@ const defaultConfig = {
     }
   },
   server: {
-    host: 'localhost',
-    proxyPort: 25566
+    targetHost: 'localhost',
+    targetPort: 25565,
+    target_server_is_online: false,
+    proxyPort: 25566,
+    online_mode: false
+  },
+  client: {
+    username: 'username/email',
+    password: 'password123'
```

Comment:
Isn't there a way to avoid setting username/email in settings?

Reply:
yea, it seems possible. But for that you need to have the `username`, `clientToken` and `accessToken`. So that means that we would need to read the `launcher_profiles.json` which is located in your `.minecraft` folder.

Reply:
Reading `.minecraft` is as bad as hell. For example multimc keeps separate `.minecraft` per instance. I'll look into the client implementation to check if there's a way to skip the credentials. For now leave it be.

Reply:
Not skipping the credentials is the whole point of adding an option to join online mode servers since thats how the server can validate or your account is real or not cracked?

What if we make the path to the launcher_profile.json as config value so you can change it. And make the creation of a client more dynamic so you can set the config after you have started up the proxy.
Or maybe just start the webserver and start the proxy after setting the file path if you want to use online mode servers?

Reply:
launcher_profile.json path as config is too much. I mean that this tool should be as simple as we can make it. Let's leave it be right now, I'll make an issue after merging it. It's not a must have to skip the credentials it's just that I don't really like an idea of putting my minecraft credentials to some tool I found on the internet. For a regular user it may (or may not) sound sketchy. 

I'm totally aware that those credentials won't leak if you host it locally. But let's be honest there is a chance that someone would use it over the net/on exposed port. In theory, after enabling the `online-mode` setting, inputting the credentials and saving, we could remove the username and password from the frontend entirely. Next time the settings will be opened, the password field will be filled with some random string, and username with (idk, maybe) `email.hidden@example.com`, both inputs disabled and a text with an open padlock icon saying `🔓 Change credentials` or something like that.

Second option, the one I'm more convinced to is to make the credentials as a server side settings. No inputs in the frontend, just in the server config file.